src/common/constants/api.js

```diff
@@ -1,2 +1,2 @@
-// eslint-disable-next-line import/prefer-default-export
 export const LINES_API = 'orders/order-lines';
+export const INSTANCE_STATUSES_API = 'instance-statuses';
```

Comment:
Now we have two files with API paths: this one and `src/components/Utils/api.js` .
Should we keep it in one place?

Reply:
we keep common code under`common` dir as a pattern in all acquisitions repos. `src/components/Utils/api.js` - is a legacy and a deprecated way. I don't see too much sense to hustle around keeping all in one place for now